contracts/BodhiToken.sol

```diff
@@ -7,11 +7,16 @@ contract BodhiToken is StandardToken, Ownable {
   // Token configurations
   string public constant name = "Bodhi Token";
   string public constant symbol = "BOT";
-  uint256 public constant decimals = 18;
+  uint256 public constant decimals = 8;
 
-  // Token constants
-  uint256 public constant saleAmount = 60 * (10**6) * (10**decimals); // 60 million BOT tokens for sale
-  uint256 public constant tokenTotalSupply = 100 * (10**6) * (10**decimals); // 100 million BOT tokens will ever be created
+  /// @notice 60 million BOT tokens for sale
+  uint256 public constant saleAmount = 60 * (10**6) * (10**decimals);
+
+  /// @notice 100 million BOT tokens will ever be created
+  uint256 public constant tokenTotalSupply = 100 * (10**6) * (10**decimals);
+
+  /// @notice Number of decimals for token being exchanged for BOT
+  uint256 public constant exchangeTokenDecimals = 8;
```

Comment:
keep this above `decimals`

Reply:
also rename `exchangeTokenDecimals` into `nativeDecimals`, because by definition, BOT itself is actually a exchange Token :)

Reply:
done